src/index.html

```diff
@@ -4,9 +4,267 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="moyo-header">
+      <img class="logo-img"
+       src="./images/logo.png"
+       alt="logo"
+     >
+      <nav class="nav">
+        <a class="nav__link is-active" href="#">Apple</a>
+        <a class="nav__link" href="#">Samsung</a>
+        <a class="nav__link" href="#">Смартфоны и телефоны</a>
+        <a class="nav__link"
+         data-qa="nav-hover"
+         href="#"
+       >
+          Ноутбуки и компьютеры
+        </a>
+        <a class="nav__link" href="#">Гаджеты</a>
+        <a class="nav__link" href="#">Планшеты</a>
+        <a class="nav__link" href="#">фото</a>
+        <a class="nav__link" href="#">видео</a>
+      </nav>
+    </header>
+    <main class="main-cards">
+      <div class="container-main-cards">
+        <div data-qa="card" class="card">
+          <div class="container">
+            <img class="card-img"
+             src="./images/imac.jpeg"
+             alt="monitor"
+           >
+            <div class="description">
+              <p class="description__name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</p>
+              <p class="description__code">Код товара: 195434</p>
+            </div>
+            <div class="block-stars">
+              <div class="stars">
+                <div class="star stars--1"></div>
+                <div class="star stars--1"></div>
+                <div class="star stars--1"></div>
+                <div class="star stars--1"></div>
+                <div class="star stars--0"></div>
+              </div>
```

Comment:
```suggestion
              <div class="stars--4">
                <div class="star"></div>
                <div class="star"></div>
                <div class="star"></div>
                <div class="star"></div>
                <div class="star"></div>
              </div>
```
Это должно выглядеть вот так